lib/fluent/plugin/out_splunk.rb

```diff
@@ -0,0 +1,313 @@
+# frozen_string_literal: true
+
+require 'fluent/output'
+require 'fluent/plugin/output'
+require 'fluent/plugin/formatter'
+require 'prometheus/client'
```

Comment:
The PR is about adding the Splunk SCP / Ingest API - this adds a new dependency and is about prometheus. altough i like additional metrics I would prefer to not mix features in one PR.